src/data/qpf/multivariate/constructions/sigma.lean

```diff
@@ -0,0 +1,107 @@
+
+/-
+Copyright (c) 2018 Simon Hudon. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Author: Simon Hudon
+-/
+
+import data.pfunctor.multivariate.basic
+import data.qpf.multivariate.basic
+
+/-!
+# Dependent product and sum of QPFs are QPFs
+
+-/
+
+universes u
+
+namespace mvqpf
+open_locale mvfunctor
+
+variables {n : ℕ}
+  {A : Type.{u}}
```

Comment:
Isn't the preferred idiom to put these all one one line or repeat the `variables` keyword?